app/src/main/java/sk/inloop/support/sample/FixedMainActivity.java

```diff
@@ -0,0 +1,56 @@
+package sk.inloop.support.sample;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.View;
+import android.widget.Button;
+
+import sk.inloop.support.sample.adapter.CorrectFragmentAdapter;
+
+public class FixedMainActivity extends AppCompatActivity {
```

Comment:
We should separate these "test" activities into a separate module - so that it's not part of the library .aar.